src/opera/pge/rtc_s1/rtc_s1_pge.py

```diff
@@ -465,23 +465,10 @@ def _collect_rtc_product_metadata(self, metadata_product):
         output_product_metadata = get_rtc_s1_product_metadata(metadata_product)
 
         # Fill in some additional fields expected within the ISO
-        output_product_metadata['frequencyA']['frequencyAWidth'] = len(output_product_metadata['frequencyA']['xCoordinates'])
-        output_product_metadata['frequencyA']['frequencyALength'] = len(output_product_metadata['frequencyA']['yCoordinates'])
-
-        # TODO: the following fields seems to be missing in the interface delivery products,
```

Comment:
I don't think this section between the `TODOs` should have been removed.

Reply:
I put them back in, but I removed them because all the fields were found so I assumed that those specific fields were now available.